core/options.hpp

```diff
@@ -144,6 +144,8 @@ struct Options
 			 "Control that determines whether the AF algorithm is to move the lens as quickly as possible or more steadily.(normal, fast)")
 			("autofocus-window", value<std::string>(&afWindow)->default_value("0,0,0,0"),
 			"Sets AfMetering to  AfMeteringWindows an set region used, e.g. 0.25,0.25,0.5,0.5")
+			("lens-position", value<std::string>(&lens_position_)->default_value(""),
+			 "Set the lens to a particular focus position, expressed as a reciprocal distance (0 moves the lens to infinity), or \"default\" for the hyperfocal distance")
```

Comment:
Should we perhaps use "hyperfocal" explicitly here instead of default? I'm not really fussed either way.

Reply:
Hyperfocal: I didn't want to pre-empt the libcamera specification so I was keeping it a bit vague, but could change this.

Reply:
Good point, maybe best leave as "default" for now.